### PR TITLE
User can review his order 

### PR DESCRIPTION
**Pull request by offcast-dev**
PT Link: https://www.pivotaltracker.com/story/show/177903773
## User Story
```
As a visitor,
In order to see my full order and a total price
I would like be able to see an order details view
```
## Changes Proposed
- navigation menu contains tab to view the whole order
- order is displayed in categories (starters, mains, etc.)
- same menu positions are counted up
- in the bottom total price is displayed

## Video for test
https://recordit.co/2A5I1pkQ5L

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
 import MenuItemList from './components/MenuItemList'
+import ViewOrder from './components/ViewOrder'
 import Registration from './components/Registration'
 import { Container, Header, Menu } from 'semantic-ui-react' 
 import LogIn from './components/LogIn'
 
 class App extends Component {
   state = {
-    category: "starters",
+    tab: "starters",
```
Comment (lukeperrin10) [inline src/App.jsx:10]: why the change to tab from category?

File: cypress/integration/userCanSeeTheOrder.feature.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+          cy.get('[data-cy="title"]').should("contain.text", "BB beans");
+          cy.get('[data-cy="amount"]').should("contain", "x1");
+          cy.get('[data-cy="price"]').should("contain", 135);
+        });
+      });
+      cy.get('[data-cy="beverages"]').within(() => {
+        cy.get('[data-cy="header"]').should("contain.text", "Beverages");
+        cy.get('[data-cy="item-0"]').within(() => {
+          cy.get('[data-cy="title"]').should("contain.text", "Coke");
+          cy.get('[data-cy="amount"]').should("contain", "x1");
```
Comment (arunbhalli) [inline cypress/integration/userCanSeeTheOrder.feature.js]: what is the meaning of x1 and x2 here?

**Review by lukeperrin10 — CHANGES_REQUESTED** [review]
If this is completed, why is it WIP?

File: src/App.jsx (hunk 1 above)
Comment (offcast-dev) [reply src/App.jsx:10]: We added View Order button to navigation menu next. So now we had to change naming to something more general that categories, to make sense.

Comment (lukeperrin10) [reply src/App.jsx:10]: Thanks for the explanation :)

File: src/components/ViewOrder.jsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  };
+
+  calculateTotalPrice = () => {
+   
+    this.state.menuData.map((item) => {
+      let totalPrice = this.state.totalPrice + item.price
+      this.setState({totalPrice: totalPrice}) 
+      
+    })
+    debugger
```
Comment (arunbhalli) [inline src/components/ViewOrder.jsx]: please remove the debugger

**Review by arunbhalli — CHANGES_REQUESTED** [review]
Total price is not easily readable for the user  
please add screen of user interface.

File: cypress/integration/userCanSeeTheOrder.feature.js (hunk 2 above)
Comment (offcast-dev) [reply cypress/integration/userCanSeeTheOrder.feature.js]: we removed in the later commits

File: src/components/ViewOrder.jsx (hunk 3 above)
Comment (offcast-dev) [reply src/components/ViewOrder.jsx]: good spot! Thank You!

**Comment by lukeperrin10** [comment]
Good commits, easily readable.

**Review by lukeperrin10 — APPROVED** [review]

**Review by arunbhalli — APPROVED** [review]
